app/assets/src/components/views/discovery/mapping/BaseMap.jsx

```diff
@@ -20,6 +20,11 @@ class BaseMap extends React.Component {
     };
   }
 
+  updateViewport = viewport => {
+    this.setState({ viewport });
+    this.props.updateViewport && this.props.updateViewport(viewport);
```

Comment:
As a note for the future (need to update dev guidelines), we going towards a pattern where we load all props and state at the beginning of the function, like

```javascript
updateViewport = viewport => {
	const { updateViewPort } = this.props;
    this.setState({ viewport });
    updateViewport && updateViewport(viewport);
};
```

Becomes more relevant for more complex functions

Reply:
👍